src/sentry/services/hybrid_cloud/organization/__init__.py

```diff
@@ -134,6 +138,7 @@ class ApiTeam:
     organization_id: int = -1
     slug: str = ""
     actor_id: int | None = None
+    org_role: str = ""
```

Comment:
i'm not sure what's important to include in ApiTeam so this can be removed if it's unnecessary